packages/synthetix-governance/contracts/interfaces/ISynthetixElectionModule.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.8.0;
 import {IElectionModule as IBaseElectionModule} from "@synthetixio/core-modules/contracts/interfaces/IElectionModule.sol";
 
 interface ISynthetixElectionModule is IBaseElectionModule {
+    /// @notice Initializes the module and immediately starts the first epoch
```

Comment:
Sorry for the noise, just bringing docs to the interface to be consistent with core-modules